hack/boilerplate/boilerplate.sh.txt

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) YEAR-present Mattermost, Inc. All Rights Reserved.
```

Comment:
YEAR-present

Reply:
this is the template to test against the files, should be like that